client/dist/main.css

```diff
@@ -6,39 +6,109 @@ aside {
   background-color: lightgrey;
 }
 
-.ratings {
+
+/* Review Section CSS */
+
+/* Rating and Product Breakdown */
+.rating-product-breakdown {
   float: left;
   padding: 20px;
+  width: 300px;
 }
 
-.review {
-  padding: 10px;
+.rating {
+  display:inline-flex;
 }
 
-#review-list {
-  padding: 10px;
-  height: 700px;
-  overflow-y: scroll;
-  scroll-snap-type: y mandatory;
-  scroll-snap-align: start;
+.rating-bar {
+  width: fit-content;
+  display: grid;
+  padding-left: 5px;
+  padding-right: 5px;
 }
 
-.question {
+#grey-bar {
+  border-color: lightgrey;
+  background-color: lightgrey;
+  width: 200px;
+  height: 5px;
+  grid-area: 1/-1;
+}
+
+#green-bar {
+  border-color: darkgreen;
+  background-color: darkgreen;
+  height: 5px;
+  grid-area: 1/-1;
+  margin-left: 0;
+}
+
+.breakdown {
+  padding-top: 10px;
+}
+
+.product-breakdown {
+  display: grid;
+  width: 300px;
+}
+
+.breakdown-bar {
+  display: inline-flex;
+  grid-area: 1/-1;
+}
+
+/* pointer midpoint is -2% of margin, so code will require percentage-2% */
+.pointer {
+  grid-area: 1/-1;
+  margin-left: 48%;
+}
+
+.productbreakdown-bar {
+  width: 100%;
+  height: 5px;
+  border-color: lightgrey;
+  background-color: lightgrey;
+  margin: 3px;
+}
+
+.stars {
+  display: grid;
+  width: fit-content;
+}
+
+#half-star {
```

Comment:
It is preferred to have classes instead of IDs. Is there a specific tool you're using which requires an ID?

Reply:
Nope, I will refactor the ID to class names instead